.github/workflows/ci.yml

```
@@ -559,37 +559,37 @@ jobs:
         set +e; source /opt/intel/oneapi/setvars.sh; set -e
         cmake --build build-11 --target test_cmake_build
 
-    - name: Configure C++14
+    - name: Configure C++17
       run: |
         set +e; source /opt/intel/oneapi/setvars.sh; set -e
-        cmake -S . -B build-14     \
+        cmake -S . -B build-17     \
```

Let's try this if we want to keep `-Werror-all`:
```suggestion
        export CXXFLAGS="${CXXFLAGS} -Wno-conversion"
        cmake -S . -B build-17     \
```